nomos-services/data-availability/sampling/src/lib.rs

```diff
@@ -291,10 +271,10 @@ where
         }
 
         // Fall back to API request
-        let (reply_sender, reply_channel) = oneshot::channel();
-        if let Err(e) = api_request.request_commitments(blob_id, reply_sender).await {
-            error!("Error sending request to API backend: {e}");
-        }
+        let (_reply_sender, reply_channel) = oneshot::channel();
+        // if let Err(e) = api_request.request_commitments(blob_id, reply_sender).await
```

Comment:
The new service message wasn't introduced in this PR because we'd need to propagate the Commitment type?